storage/zynq7000-flash/qspi.c

```diff
@@ -382,16 +393,21 @@ static int qspi_initPins(void)
 {
 	int res, i;
 	static const int pins[] = {
-		pctl_mio_pin_01, /* Chip Select */
-		pctl_mio_pin_02, /* I/O */
-		pctl_mio_pin_03, /* I/O */
-		pctl_mio_pin_04, /* I/O */
-		pctl_mio_pin_05, /* I/O */
-		pctl_mio_pin_06, /* CLK */
-		pctl_mio_pin_08  /* Feedback CLK */
+		QSPI_CS,
+		QSPI_IO0,
+		QSPI_IO1,
+		QSPI_IO2,
+		QSPI_IO3,
+		QSPI_CLK,
+		QSPI_FCLK
```

Comment:
Shouldn't we translate these defines (integers) to pctl_mio_pins somehow?

Reply:
I think that we decided that there should no be any translation. Look into other drivers which use `board_config.h`

Reply:
OK